.ado/templates/prep-and-pack-nuget.yml

```diff
@@ -166,5 +166,5 @@ steps:
           binariesToSign: |
```

Comment:
Parameters in this file, like `packMicrosoftReactNativeProjectReunion` should be renamed.

Reply:
Also should replace `outputPackage: Microsoft.ReactNative.ProjectReunion`, to set the new name of the NuGet packag to be published.

The NuSpec should also be renamed.

Reply:
done.